### PR TITLE
[SDK/CLI] Support user to configure AzureCliCredential for pfazure commands

### PR DESCRIPTION
**Pull request by 0mza987**
# Description

- [SDK/CLI] Support configuring environment variable to directly use `AzureCliCredential` for `pfazure` commands.
  ```dotenv
  USE_AZURE_CLI_CREDENTIAL=true
  ```

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 1.6.0 (TBD)
+
+### Features Added
+
+- [SDK/CLI] Support configuring environment variable to directly use `AzureCliCredential` for `pfazure` commands.
+  ```dotenv
+  USE_AZURE_CLI_CREDENTIAL=true
```
Comment (wangchao1230) [inline src/promptflow/CHANGELOG.md:9]: our envrionmetn should starts with PF_

Comment (0mza987) [reply src/promptflow/CHANGELOG.md]: good point, updated

File: src/promptflow/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 1.6.0 (TBD)
+
+### Features Added
+
+- [SDK/CLI] Support configuring environment variable to directly use `AzureCliCredential` for `pfazure` commands.
```
Comment (wangchao1230) [inline src/promptflow/CHANGELOG.md:7]: [SDK/CLI] -> [CLI]? as this only appies for CLI

**Review by wangchao1230 — APPROVED** [review]

**Review by Stephen1993 — APPROVED** [review]

File: src/promptflow/CHANGELOG.md (hunk 2 above)
Comment (0mza987) [reply src/promptflow/CHANGELOG.md:7]: well lets change it in the next pr

**Pull request merged**